setup.py

```diff
@@ -25,13 +25,13 @@
     "nose-focus",
     "colorama",
     "numpy",
-    "scipy>=0.16.1,<1.0.0",
     "flake8",
     "yanc",
     "remotecv",
-    "pyssim",
+    "pyssim>=0.4.0",
```

Comment:
I'm wondering why we're setting a lower threshold for `pyssim` here. There's nothing different in tests nor in docs on this commit. I looked briefly but could not see any relation between pyssim >= 0.4 and latest Python versions. Excuse me if I missed something.

Reply:
so, this is only a list of test dependencies. 
previously, scipy was manually included, but never used directly anywhere in the code. Only usage was through pyssim. 
Pyssim, in it's turn, does not seem to install its dependencies automatically, up to latest release (0.4). https://github.com/jterrace/pyssim/issues/24
Now that pyssim can automatically install it's own dependencies, I decided to remove explicit require of `scipy`, as it was not directly used and let pip's pyssim do the magic.

Reply:
Oh, my bad, didn't notice that diff regarded changes made to `test_require`.